### ISSUE TITLE
Hakai Oceanography Chlorophyll datasets

### ISSUE THREAD

**Issue by JessyBarrette** [HakaiInstitute/metadata-review#71]
This issue summarizes all the different datasets related to Hakai Chlorophyll data product to be completed.

- [x] #69
- [x] #68 
- [ ] Hakai Chlorophyll-a samples Research https://cioos-siooc.github.io/metadata-entry-form/#/en/hakai/7U7b8oPpeTN6gjvXlUCTGJr5pga2/-MadZvZJ4ZeikpCgRcV8
- [ ] Hakai Chlorophyll-a Provisional https://cioos-siooc.github.io/metadata-entry-form/#/en/hakai/7U7b8oPpeTN6gjvXlUCTGJr5pga2/-McQFPAf457LB4-SWmyL
- [ ] Hakai HPLC https://cioos-siooc.github.io/metadata-entry-form/#/en/hakai/7U7b8oPpeTN6gjvXlUCTGJr5pga2/-MadZvZJ4ZeikpCgRcV8

**Comment by jdelbel** [comment]
I cross checked the metadata records for the chlorophyll research and provisional datasets. I 1) reviewed the forms and think that the records are good as is; although, it would be good to make some tweaks; 2) I removed Jen Jackson from the contacts; 3) updated some of the roles for myself and Bryn and 4) generated draft DOIs.

**Comment by JessyBarrette** [comment]
Looks like you did publish the Chlorophyll research dataset metadata record.

**Comment by timvdstap** [comment]
Hey @jdelbel! I hope you had an amazing Christmas! I'm wondering if I can close this issue, to me it seems like the records in this issue are no longer active:

- Hakai Chlorophyll-a samples Research
- Hakai Chlorophyll-a Provisional
- Hakai HPLC

Should new metadata records for these be created?

**Comment by timvdstap** [comment]
Hey @jdelbel - are you OK with me closing this issue? 🦖

**Comment by fostermh** [comment]
it appears there is a provisional dataset in our erddap 

https://catalogue.hakai.org/erddap/tabledap/HakaiChlorophyllSampleProvisional.html

and a metadata record for it here

https://catalogue.hakai.org/dataset/ca-cioos_1755dc37-8d33-4158-8041-c22536fd5771

**Comment by fostermh** [comment]
there is also an hplc record
https://catalogue.hakai.org/dataset/ca-cioos_7fd3ec6c-083a-4942-84b4-215e69492072
https://catalogue.hakai.org/erddap/tabledap/HakaiHPLCResearch.html

**Comment by fostermh** [comment]
so is the problem just that we don't have an ERDDAP `Hakai Chlorophyll-a samples Research` entry?

**Comment by timvdstap** [comment]
@fostermh Yeah I believe that's what prompted me to revisit this issue, I couldn't find a Hakai Chlorphyll-a sample research dataset in ERDDAP (was prompted through the link checker). 

> Looks like you did publish the Chlorophyll research dataset metadata record.
> 
> See https://catalogue.hakai.org/dataset/ca-cioos_d55021c3-a142-4e14-8208-36c9826c1893

^ERDDAP link in this metadata record doesn't point to anything.

**Comment by fostermh** [comment]
the issue is that there are still no chlorophyll data that is 'research-ready'. The view in the database still returns zero rows so the dataset in erddap has not been created.

**Comment by fostermh** [comment]
the query that populates the reasearch ready dataset is very odd however. it is limited to a very small number of sites, only ones with quality_level of 'publication' which doesn't exist, and only data after 2018-05-04

**Comment by jdelbel** [comment]
That's on me -  Although, I thought we had worked on this and re-flagged. I potentially didn't change a specific column. I was pretty conservative on this when I was working with Jessy and suggested quite a few thresholds for pushing to this data set.

I can take a look and do some re-flagging to see if it pushes data.

**Comment by fostermh** [comment]
ok, the provisional dataset does not have these restrictions which is why it seems odd. I would have expect the only difference to be the quality level. If set the cut off as a quality level of 'Principal Investigator' and remove the date cutoff we get around 2000 rows

**Comment by jdelbel** [comment]
@fostermh Just saw that last message after I sent my previous. Yes... it really depends on how you qualify "research ready" - We made some big changes to our process/methods once I started digging into these data and anything pre-2018-05-04 has some question marks. The data follow expected trends, but may have %error associated that is hard to quantify. Also, I've only ever really deeply investigated core station data - that is why we limited stations. Although I'm more confident pushing data from other stations now that we have improved our methods.

**Comment by jdelbel** [comment]
Another consideration is that that prior to the last few years we weren't diluting high concentrations samples - data "seem" OK, but I have flagged them as ADL. Hence they should only go to the provisional.

**Comment by timvdstap** [comment]
> That's on me - Although, I thought we had worked on this and re-flagged. I potentially didn't change a specific column. I was pretty conservative on this when I was working with Jessy and suggested quite a few thresholds for pushing to this data set.
> 
> I can take a look and do some re-flagging to see if it pushes data.

I think you might be referring to #106 ?

Regarding your comment

> Yes... it really depends on how you qualify "research ready"

I'm wondering what the best next steps are - are the qualifications for research ready data still under consideration? Or do we already know these and we 'just' have to make some changes to the code to push the data to ERDDAP? Should the link to the ERDDAP dataset for now be removed, or the metadata record put back into draft?

**Comment by fostermh** [comment]
I think we need to have a real time chat. if we keep the date and site restrictions but lower the quality level to PI then we get 14 rows returned currently. @jdelbel you know best what is 'Research Ready' I just think we need to streamline the workflow and query a bit.

**Comment by timvdstap** [comment]
I'll reach out to you both over Slack!

**Comment by timvdstap** [comment]
Summary from last week's meeting:

- Quality level set to 'Principal Investigator' for research ready data -- link to ERDDAP should work next time data is pushed there.
- Justin to flag additional entries (hurray for 'flaggin' Friday'!)

Things to follow up on:
- Move QAQC scripts to a Hakai GitHub repository
- Discuss: differences in Provisional datasets across records - some only show the last 3 months of data, others the entire dataset. Does it make sense to have this be consistent practice across Hakai for all data types or is there reasons for different time restrictions?

**Comment by timvdstap** [comment]
Closing this issue, the final [record](https://catalogue.hakai.org/dataset/ca-cioos_d55021c3-a142-4e14-8208-36c9826c1893) that was discussed here is published and points to a Research-ready dataset.
